src/components/MatchesTable.vue

```diff
@@ -140,11 +174,11 @@ export default {
           (match.cancelled == 0 || match.cancelled == null) &&
           match.start_time != null
         ) {
-          matchString = `Live, ${team1Score}:${team2Score} vs ${match.team2_string}`;
+          matchString = `Live, ${team1Score}:${team2Score} - ${match.team1_string} vs ${match.team2_string}`;
```

Comment:
Oh I see what you mean. This was a design choice since both teams are already showing in the table. So Team 1 is always considered the "Home" team, so this is why it was always `vs Team2`. I'm not opposed, but how does this look/feel on smaller screens or mobile layout?

Reply:
Like this from my browser with mobile compatiblity
![image](https://user-images.githubusercontent.com/16237642/201493923-74bc8f34-542e-412f-a28f-1c8c5d2db6a5.png)